core/video_options.hpp

```diff
@@ -58,6 +58,10 @@ struct VideoOptions : public Options
 			 "Write output to a circular buffer of the given size (in MB) which is saved on exit")
 			("frames", value<unsigned int>(&frames)->default_value(0),
 			 "Run for the exact number of frames specified. This will override any timeout set.")
+			("metadata", value<std::string>(&metadata),
+			 "Save captured image metadata to a file or \"-\" for stdout")
+			("metadata-plain", value<bool>(&metadata_plain)->default_value(false)->implicit_value(true),
+			 "Save the metadata as plaintext rather than json (requires --metadata)")
```

Comment:
Was this going to become "metadata-format"? I think the "still" version has that, so probably make this one the same. Maybe this option just vanishes from here in favour of one in the base class.

Reply:
Yes, sorry, that's fixed in the second commit